app/jobs/thredded/activity_updater_job.rb

```diff
@@ -4,10 +4,10 @@ class ActivityUpdaterJob < ::ActiveJob::Base
     queue_as :default
 
     def perform(user_id, messageboard_id)
-      now = Time.zone.now
+      now = Time.current
 
-      user_detail = Thredded::UserDetail.for_user_id(user_id)
-      user_detail.update_column(:last_seen_at, now)
+      user_detail = Thredded::UserDetail.where(user_id: user_id).first_or_initialize
+      user_detail.update!(last_seen_at: now)
```

Comment:
`update!` will save the object if it's a new record and update it if it's an existing record?